reference/tax_provider.yml

```diff
@@ -961,6 +966,23 @@ components:
         - quantity
         - type
       x-internal: false
+    request-item-tax-property:
+      type: object
+      description: A simple key value pairing allowing merchants to provide an additional input into a tax provider's tax calculation.
+      title: TaxProperty
+      properties:
+        code:
+          type: string
+          description: Used by tax providers to programmatically identify a specific calculation input.
+          example: alcohol-percentage
+        value:
+          type: string
+          description: The value that will be factored into the tax provider's tax calculation rules, where supported.
+          example: '4.9'
+      required:
+        - code
+        - value
+      x-internal: false
```

Comment:
@bc-andreadao @slsriehl I noticed a bunch of these have been added more recently, so I've added it here too - does that look correct? Cheers.

Reply:
Yes, we're setting internal flags to false and allowing models to show to avoid schema problems